### PR TITLE
Update docs - issues guide with new rules

### PR DESCRIPTION
**Pull request by karelz**
Update docs - issues guide with new rules (labels, area labels, assignments, area owners).

cc: @danmosemsft @weshaggard

### CODE REVIEW

File: Documentation/project-docs/issue-guide.md [hunk 1]
```diff
@@ -6,33 +6,90 @@ This page outlines how the CoreFx team thinks about and handles issues.  For us,
...
+| [System.Security](https://github.com/dotnet/corefx/labels/area-System.Security)               |  |  |  |
+| [System.ServiceModel](https://github.com/dotnet/corefx/labels/area-System.ServiceModel)       |  |  |  |
+| [System.ServiceProcess](https://github.com/dotnet/corefx/labels/area-System.ServiceProcess)   |  |  |  |
+| [System.Text.Encoding](https://github.com/dotnet/corefx/labels/area-System.Text.Encoding)     |  |  |  |
+| [System.Text.RegularExpressions](https://github.com/dotnet/corefx/labels/area-System.Text.RegularExpressions) | @priya91 |  |  |
+| [System.Threading](https://github.com/dotnet/corefx/labels/area-System.Threading)             | @kouvel, @alexperovich |  | Triaged |
+| [System.Transactions](https://github.com/dotnet/corefx/labels/area-System.Transactions)       |  |  |  |
+| [System.Xml](https://github.com/dotnet/corefx/labels/area-System.Xml)                         | @sepidehMS, @krwq  | | Triaged |
+| [System](https://github.com/dotnet/corefx/labels/area-System)                                 |  | Issues without clear contract association (corss-contract, or potentially new contracts) |  |
```
Comment (danmosemsft) [inline Documentation/project-docs/issue-guide.md]: typo corss

File: Documentation/project-docs/issue-guide.md [hunk 2]
```diff
@@ -6,33 +6,90 @@ This page outlines how the CoreFx team thinks about and handles issues.  For us,
...
+
+### Milestones
+We use [milestones](https://github.com/dotnet/corefx/milestones) to prioritize work for each upcoming release to NuGet.org.
 
 ### Assignee
-We assign each issue to a CoreFx team member.  In most cases, the assignee will not be the one who ultimately fixes the issue (that only happens in the case where the issue is tagged "grabbed by assignee"). The purpose of the assignee is to act as a point of contact between the CoreFx team and the community for the issue and make sure it's driven to resolution.  If you're working on an issue and get stuck, please reach out to the assignee (just at mention them)  and they will work to help you out.
+We assign each issue to assignee, when the assignee is ready to pick up the work and start working on it.  If the issue is not assigned to anyone and you want to pick it up, please say so - we will assign the issue to you.  If the issue is already assigned to someone, please coordinate with the assignee before you start working on it.
+
+Note: This is a new scheme introduced in 2016/9.  The old scheme used assignees as area owners.  We will migrate issues to the new scheme throughout 2016.
```
Comment (danmosemsft) [inline Documentation/project-docs/issue-guide.md]: I don't think you need the history

**Comment by danmosemsft** [comment]
Could you add all the owners, even for untriaged areas? There really is no reason not to.

File: Documentation/project-docs/issue-guide.md [hunk 3]
```diff
@@ -6,33 +6,90 @@ This page outlines how the CoreFx team thinks about and handles issues.  For us,
...
+
+### Areas
+Areas are tracked by labels area-* (e.g. area-System.Collections), or Infrastructure. Each area typically corresponds to one or more contract assemblies.
+
+| Area                                                                                          | Owners / experts | Description | Triage Status |
```
Comment (karelz) [inline Documentation/project-docs/issue-guide.md]: It captures which areas have been already triaged (and therefore Assignee matters). When all areas are triaged, we can delete the column.

File: Documentation/project-docs/issue-guide.md (hunk 2 above)
Comment (karelz) [inline Documentation/project-docs/issue-guide.md]: I think we need the note for the time we are in transition ...

**Comment by karelz** [comment]
> Could you add all the owners, even for untriaged areas?

I'd like to check with the owners first, before I put them up :). I can make it high pri task after this doc update, not blocking the changes. Would that work?

File: Documentation/project-docs/issue-guide.md (hunk 1 above)
Comment (karelz) [inline Documentation/project-docs/issue-guide.md]: Done

**Pull request merged**